Calc.java

turkanaaa opened the pull request:
Write a program that takes two integers from command-line arguments and prints the sum, subtraction, multiplication, integer division and remainder of the division of those two numbers.

Commit:
Calc.java

Write a program that takes two integers from command-line arguments and
prints the sum, subtraction, multiplication, integer division and remainder of the division of
those two numbers.

Kunal-debug-png commented:
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {

        int a,b;
        Scanner sc=new Scanner(System.in);
        a=sc.nextInt();
        b= sc.nextInt();
        System.out.println("Addition "+(a+b));
        System.out.println("Subtraction "+(a-b));
        System.out.println("Mulitipication "+(a*b));
        System.out.println("Divison "+(a/b));
        System.out.println("Remainder "+(a%b));

    }
}


here is the possible solution ;)

Commit:
Create EX

Commit:
Create PW

Commit:
Calc.java

Write a program that takes two integers from command-line arguments and
prints the sum, subtraction, multiplication, integer division and remainder of the division of
those two numbers.

Commit:
Time.java

Write a program that given a number that represents seconds, converts it to
hours, minutes, and seconds. (Use Scanner or command-line arguments).

Commit:
PolarityOfNumber.java

Given a double-precision floating-point number. Check and print its polarity (negative, positive, neutral = 0).

Commit:
MonthInfo.java

Given the number of the month (e.g., 1 = Jan, 2 = Feb, etc.), print the name of the month followed by the number of days in it.

Commit:
LeapYear.java

Given a year. Write a program to decide whether it is a leap year or not.

Commit:
Fact.java

Write a program that takes an integer using command-line arguments and find
the factorial of that number.

Commit:
SumOfOdd.java

Given two integers, find the sum of the odd numbers in between those
numbers. Note: the order of the numbers is not guaranteed.

Commit:
Delete PW

Commit:
Delete EX

Commit:
Average.java

Given numbers (the count is not known in advance, take inputs until the
EOF) and find the average of these numbers.

Commit:
Harmonic.java

Given an integer n. Find the nth harmonic number. Note that nth harmonic number is the nth element of the harmonic series.

Commit:
Delete Fact.java

Commit:
Fact.java

Write a program that takes an integer using command-line arguments and find
the factorial of that number.

Commit:
Digits.java

Given an integer (can be negative as well). Find the sum, product, and average
of its numbers.

Commit:
CoinTossing.java

Write a program to simulate a coin tossing (for simplicity, take fair coin
with 50/50 chance of showing head/tail when it lands). Use Math.random() method.

Commit:
RollingDie.java

Write a program to simulate the roll of a loaded six-sided die, where
the values 1, 2, 3, 4 appear with probability 1/8 and the values 5 and 6 appears
with probability 1/4 and print the resulting number.

Commit:
PrintingPatterns.java

Print the following pattern given the size: (the following example size = 5).
*#*#* 
#*#*# 
*#*#* 
#*#*#

Commit:
RandomPointsInCircle.java

Print a point chosen uniformly at random inside the circle of radius 1 centered at (0, 0).

Commit:
MinMax.java

Given floating-point values (the count is not known in advance, take inputs
until the EOF), find the minimum and the maximum of the numbers.

Commit:
Primes.java

Given an integer. Check if the number is prime.

Commit:
ArmstrongNumbers.java

Given an integer. Check if the number is a Narcissistic number (Armstrong number) or not.
a. Example: Let us assume the number is 371. To check if the number is an Armstrong number, we should raise each digit to the power of 3 (number of the its digits). 3^3 + 7^3 + 1^3 = 371. If the sum equals to the number itself, it is considered to be an Armstrong number.
b. If user enters two integers, can you print out the list of all such numbers in a given range?

Commit:
TaylorSin.java

Given a floating-point value x and a positive integer n. Find the sin(x) by applying Taylor series for sine and use n as the number of terms to be accumulated.

Commit:
DiceMan.java

Write a program to simulate a game, “Dice Man”. The user (player) is asked to suggest a number that can be obtained by throwing two dice (a 6-sided fair dice) and then computer “throws two dice”. If the result is equal to what the user suggested, player wins. Otherwise, computer wins. The game is played in rounds:
a. program is terminated if user enters 0
b. prints an error message if the user enters a value which is impossible to obtain with two
dice.

Commit:
GuessGame.java

Write a program to simulate the “Guess Game”. Your program takes a
positive random integer number not greater than A (to be entered by the user). It then lets user to guess the number. If the number is guessed correctly, user wins. Otherwise, the program helps user by indicating whether the number entered is LOWER or HIGHER.

Commit:
Invoice.java

Commit:
CustomDate.java

Commit:
CustomTime.java

Commit:
ComplexNumber.java

Commit:
Point.java

Commit:
Segment.java

Commit:
Add files via upload

Commit:
Add files via upload

Commit:
Add files via upload

Commit:
Add files via upload

Commit:
Add files via upload

Commit:
Add files via upload

Commit:
Add files via upload
